gtfs/spec/en/reference.md

```diff
@@ -413,6 +419,85 @@ Trip planners normally calculate transfer points based on the relative proximity
+|  from_stop_id | **Required** | The **from_stop_id** field contains a stop ID that identifies a stop, entrance or node where the pathway begins. Stop IDs are referenced from the [stops.txt](#stopstxt) file. |
+|  to_stop_id | **Required** | The **to_stop_id** field contains a stop ID that identifies a stop, entrance or node where the pathway ends. Stop IDs are referenced from the [stops.txt](#stopstxt) file. |
+|  pathway_mode | Optional | The **pathway_mode** field specifies the type of pathway between the specified (from_stop_id, to_stop_id) pair. Valid values for this field are: |
+|   |  | * **0** or **(empty)** - Generic link to describe an undefined infrastructure (default). Not recommended. |
+|   |  | * **1** - Walkway. |
+|   |  | * **2** - Stairs. |
+|   |  | * **3** - Moving sidewalk / travelator. |
+|   |  | * **4** - Escalator. |
+|   |  | * **5** - Elevator. |
+|   |  | * **6** - Fare gate / payment gate. A pathway that crosses into an area of the station where a proof of payment is required (usually via a physical payment gate). Fare gates may either separate paid areas of the station from unpaid ones, or separate different payment areas within the same station from each other. This information can be used to avoid routing passengers through stations using shortcuts that would require passengers to make unnecessary payments, like directing a passenger to walk through a subway platform to reach a busway. |
+|   |  | * **7** - Exit gate. Indicates a pathway exiting an area where proof-of-payment is required into an area where proof-of-payment is no longer required. |
+|  pathway_type | Optional | The **pathway_type** field specifies the type of connection. Valid values for this field are: |
+|   |  | * **1** - Connects station locations to an entrance. |
+|   |  | * **2** - Connects stops/stations for transfers off the streets or within stations. |
+|   |  | * **3** - Connects stops/stations/entrances for transfers on the streets. |
+|  traversal_time | Optional | The **traversal_time** field specifies the time needed to go through the pathway on foot. Valid values for this field are: |
+|   |  | * **(empty)** - Unknown travel time. |
+|   |  | * **(non-negative integer)** - Number of seconds needed to travers this pathway on foot. |
+|  wheelchair_traversal_time | Optional | The **wheelchair_traversal_time** field specifies the time needed to go through the pathway in a wheelchair. Valid values for this field are: |
+|   |  | * **(empty)** - Unknown travel time. |
+|   |  | * **-1** - Pathway not wheelchair accessible. |
+|   |  | * **(non-negative integer)** - Number of seconds needed to travers this pathway in a wheelchair. |
+|  ramp_slope | Optional | The **ramp_slope** field specifies the slope ratio of the pathway. Valid values for this field are: |
```

Comment:
How exactly is `ramp_slope` calculated?  Is is the steepest slope between stops?

After some discussion with @astein6 who lead a research center on accessible transportation it sounds like this may need to be split into two values - steepest incline and decline, or perhaps positive and negative elevation change.

Here's how Google shows elevation change in walking directions:

![image](https://user-images.githubusercontent.com/928045/44220803-a3f38780-a14d-11e8-8e89-b41d6067085b.png)

Do we have any producers providing the slope data yet?  I'm interested to see how this translates to real data.

Reply:
True.

My assumption was that Pathways would only describe paths inside a human-made construction, and that therefore they wouldn't be any complex slope.

But now I'm thinking about those \_/ slopes you find when tunnel go under tracks... Either we modelize them with two distinct ramps, or we need indeed two values.

Do you have an example where this could be needed, knowing that this is just for a pathway and not for a full 40min journey?

Reply:
@LeoFrachet I (unfortunately) don't have an exact real-world example off the top of my head, which is why I was curious to see real producer data for slope values.